app/controllers/usergroups_controller.rb

```diff
@@ -5,7 +5,8 @@ class UsergroupsController < ApplicationController
   breadcrumb 'Groepen', :usergroups_path
 
   def index
-    @usergroups = Usergroup.all
+    @usergroups = Usergroup.not_archived
+    @archived_usergroups = Usergroup.archived
```

Comment:
Je zou dit `@usergroups = Usergroup.all` kunnen laten en dan in de view @usergroups.archived of @usergroups.not_archived. Je krijgt uiteindelijk wel dezelfde queries omdat active record collections lazy laden maar misschien mooier.